__tests__/api.test.js

```diff
@@ -45,4 +45,33 @@ describe('api/reviews', ()=>{
             })
         })
     })
+})
+
+describe('api/review/:review_id', ()=>{
+    test('responds with 200', ()=>{
+        return request(app).get('/api/reviews/1').expect(200)
+    })
+    test('responds with a single review by id', ()=>{
+        return request(app).get('/api/reviews/1').expect(200).then((review)=>{
+            const output = review.body.review[0]
```

Comment:
ideally, as we're dealing with a small set of test data, it's more secure to test for the exact data match on this review id.